bids/analysis/analysis.py

```diff
@@ -345,6 +345,12 @@ def get_design_matrix(self, names=None, format='long', mode='both',
                 be converted to dense variables and included in the returned
                 design matrix in the .dense attribute. The force argument is
                 ignored entirely if mode='both'.
+            sampling_rate ('TR', 'highest' or float): Sampling rate at which to
```

Comment:
Looks like this is working, and the only lines not getting covered are the cases where we report finding multiple or no TRs.

I think the main question is whether we like these options. I think `TR` and `float` are good, but not sure if `highest` is a good name. It's equivalent to `None`, but I thought that `None` was an odd way of specifying it, so that's why I put it in, but open to other names/approaches.

We may also want to propagate these options up to `Step.get_design_matrix`, which is the API more people will be directly using.